### DISCUSSION TITLE
Suggestion while reading the file

### DISCUSSION THREAD

**Discussion by arit20y** [koushikkothagal/ipl-dashboard#10] [Ideas]
In this method while we are reading the data. FlatFileItemReader<MatchInput> reader() we encountered an error while you had to remove the column header from the csv file as Id is unable to parse because its a column name. So if we add something like 
.resource(new ClassPathResource("match-data.csv")).delimited().names(FIELD_NAMES)
 .linesToSkip(1)

we need not to explicitly delete the column header from the raw file. As to understand the pojo we should first see the data set.

**Comment by koushikkothagal** [comment]
Thanks for calling this out! You are right, skipping the first line is another way to fix that error. Feel free to send me a PR (with this change and the first line in the csv file put back)!
